tests/ideas/test_filters.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from apps.ideas import filters, models
+
+
+@pytest.mark.django_db
+def test_idea_list_view(idea_sketch_factory, proposal_factory):
+    idea_sketch_factory()
+    idea_sketch_factory()
+    proposal_factory()
+
+    idea_filter_set = filters.IdeaFilterSet(query_data={})
+
+    filter_idea = (idea_filter_set.
+                   what_status(models.Idea.objects.all(),
+                               'some_name', 'idea_sketch')
+                   )
+    assert filter_idea.count() == 2
```

Comment:
What do thing about the following assertion: 

```python 
assert list(filter_idea) == list(filter(  
        lambda p: p.type=='Idea Sketch', models.Idea.objects.all()  ))
```
This allows testing by using something like `for type, param in ALL_POSSIBLE_TYPES` and `ALL_POSSIBLE_TYPES` is `[('Idea Sketch', 'idea_sketch'), ..]`.